CMakeLists.txt

```diff
@@ -903,6 +903,12 @@ else ()
   set ( implibname "fluidsynth" )
 endif ()
 
+if (BUILD_SHARED_LIBS)
+    set(target-type shared)
+else ()
+    set(target-type static)
+endif ()
```

Comment:
Wouldn't this redefine `target-type`? How about moving this definition a few lines up before `add_subdirectory ( src )` and removing the extra definition from `src/CMakeLists`?

Reply:
done, thanks